convert2rhel/toolopts.py

```diff
@@ -179,6 +197,13 @@ def _register_options(self):
             " 'subscription-manager orgs'. From the listed pairs"
             " Name:Key, use the Key here.",
         )
+        group.add_option(
+            "-c",
+            "--config-file",
+            help="Configuration file containing"
+            " password or activation key for subscription-manager."
```

Comment:
```suggestion
            help="A configuration file to safely provide either a user password or an activation key for registering"
                 " the system through subscription-manager. Alternatively, passing these values through the"
                 " --activationkey or --password option would leak them through a list of running processes."
```

To the description of the password and activation key, add something like: 
_"We recommend using the --config-file option instead to prevent leaking the password/activation key through a list of running processes."_

Also remove the mention of `--password-from-file` from the password option description.